### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by nunorafaelrocha**
This improves tests by adding `SeegnoTestBundle` and faker to all test classes.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,9 +20,9 @@
     },
     "require-dev": {
         "codeclimate/php-test-reporter": "~0.1.2",
-        "fzaninotto/faker": "dev-master",
         "phpdocumentor/phpdocumentor": "~2.8.1",
-        "phpunit/phpunit": "~4.4.1"
+        "phpunit/phpunit": "~4.5",
+        "seegno/test-bundle": "^0.0.1"
```
Comment (ruimarinho) [inline composer.json]: Using ^ instead of ~ here?

File: test/Bitreserve/Tests/Unit/BitreserveClientTest.php [hunk 2]
```diff
@@ -380,9 +400,9 @@ public function shouldReturnAuthorizedUserWhenResponseReturnsBearerToken()
      */
     public function shouldCreateToken()
     {
-        $login = $this->faker->userName;
-        $password = $this->faker->password;
-        $description = $this->faker->sentence;
+        $login = $this->getFaker()->userName;
```
Comment (joaonice) [inline test/Bitreserve/Tests/Unit/BitreserveClientTest.php]: Sort.

File: test/Bitreserve/Tests/Unit/BitreserveClientTest.php [hunk 3]
```diff
@@ -415,10 +441,10 @@ public function shouldCreateToken()
...
     public function shouldCreateTokenWithOTP()
     {
-        $login = $this->faker->userName;
-        $password = $this->faker->password;
-        $description = $this->faker->sentence;
-        $otp = $this->faker->randomNumber(6);
+        $login = $this->getFaker()->userName;
+        $password = $this->getFaker()->password;
+        $description = $this->getFaker()->sentence;
+        $otp = $this->getFaker()->randomNumber(6);
```
Comment (joaonice) [inline test/Bitreserve/Tests/Unit/BitreserveClientTest.php]: Sort.

File: test/Bitreserve/Tests/Unit/Paginator/PaginatorTest.php [hunk 4]
```diff
@@ -4,11 +4,12 @@
 
 use Bitreserve\Paginator\ArrayPaginator;
```
Comment (joaonice) [inline test/Bitreserve/Tests/Unit/Paginator/PaginatorTest.php]: `ArrayPaginator` is not being used

File: test/Bitreserve/Tests/Unit/Model/UserTest.php [hunk 5]
```diff
@@ -4,11 +4,12 @@
 
 use Bitreserve\BitreserveClient;
```
Comment (joaonice) [inline test/Bitreserve/Tests/Unit/Model/UserTest.php]: `BitreserveClient` is not being used.

**Pull request merged**